### PR TITLE
Reassign contact

### PR DESCRIPTION
**Pull request by amsmyth1**
### What Changed?
- Happy and Sad path for reassigning a role/contact for an incident
### What's Next?
- Refactoring and edge cases
### Known Issues?
- none
### Test Coverage?
- 92%
### README & Documentation
- Updated api endpoint
### Closed Issues
Close #55

### CODE REVIEW

File: app/controllers/api/v1/incident_contacts_controller.rb [hunk 1]
```diff
@@ -18,6 +18,26 @@ def assign_contact_to_incident
     end
   end
 
+  def reassign_role_to_different_contact
+    incident = Incident.where(id: incident_contact_params[:incident_id])
+    contact = Contact.where(id: incident_contact_params[:contact_id])
+    if incident.empty?
+      render json: {error: "Incident does not exist"}, status: :not_found
+    elsif contact.empty?
+      render json: {error: "Contact does not exist"}, status: :not_found
```
Comment (jmichlitsch) [inline app/controllers/api/v1/incident_contacts_controller.rb:27]: Love the error handling, if we have the time, maybe refactor into a error serializer or controller

**Review by jmichlitsch — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -687,6 +688,31 @@ The following is a depiction of our Database Schema
     ]
 }
 ```
+### Rassign a Contact to a Role
```
Comment (klatour324) [inline README.md:692]: Typo here: `Reassign`

**Review by klatour324 — APPROVED** [review]
Looks phenomenal. Small spelling typo. Otherwise this looks great. Lots of detail in the testing especially with the sad path specs. Great work!
